lib/geocoder/lookups/google_places_details.rb

```diff
@@ -33,9 +33,29 @@ def results(query)
         result
       end
 
+      def fields(query)
+        if query.options.has_key?(:fields)
+          return format_fields(query.options[:fields])
+        end
+
+        if configuration.has_key?(:fields)
+          return format_fields(configuration[:fields])
+        end
+
+        nil  # use Google Places defaults
+      end
+
+      def format_fields(*fields)
+        flattened = fields.flatten.compact
+        return if flattened.empty?
+
+        flattened.join(',')
+      end
+
```

Comment:
This repeats code in `GooglePlacesSearch`; let me know if you think these methods should be abstracted out to a base `class GooglePlaces < Google`. I felt it was arguable whether this was enough to justify it.